src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordWithOcrTest.java

```diff
@@ -47,22 +55,66 @@
     private static final String RESPONSE_FIELD_WARNINGS = "warnings";
     private static final String RESPONSE_FIELD_DATA = "data";
     private static final String ATTACH_TO_CASE_REFERENCE_FIELD_NAME = "attachToCaseReference";
+    private static final String SERVICE_AUTHORIZATION_HEADER = "ServiceAuthorization";
+    private static final String BEARER = "Bearer";
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Static final variables are by convention all uppercase, as long as they represent a value. This mapper is a tool, so I'd suggest `mapper`.